zanata-war/src/main/resources/messages.properties

```diff
@@ -290,6 +290,8 @@ jsf.languageSettings.LanguagesDisabled=Disabled languages: {0}.
 jsf.languageSettings.LanguageEnabled=Language "{0}" has been enabled.
 jsf.languageSettings.LanguagesEnabled=Enabled languages: {0}.
 jsf.project.LanguageUpdateFromGlobal=Updated languages from global settings.
+jsf.project.AboutPageUpdated=About page updated.
+jsf.project.AboutPageUpdateFailed=There was a problem while updating the about page.
```

Comment:
@djansen-redhat yes - basically if that message is displayed, the state is unknown to the code.